api/graphql/schema.graphql

```diff
@@ -1388,6 +1393,17 @@ input UpdateApplicationInput {
   educationRequirementWorkExperiences: UpdateEducationExperiencesRequirementBelongsToMany
 }
 
+input TeamRoleHasMany {
+  attach: TeamRolesInput
+  detach: TeamRolesInput
+  sync: TeamRolesInput
+}
+
+input AssignUsersToTeamInput {
```

Comment:
I find it a little surprising to see this structured as a batch operation.  Are we expecting to need to update groups of users to groups of roles in the UI?  Doing this as a single update would sidestep the need to use a transaction, too.

Reply:
True, we don't have any designs for UI yet which would require batch updating, I just thought it easy enough to build in the extra flexibility.